spec/unit/target/pod_target_spec.rb

```diff
@@ -634,6 +634,11 @@ module Pod
           @pod_target.uses_swift?.should == true
         end
 
+        it 'returns that it uses swift with a pre built vendored framework and specifying swift_versions' do
```

Comment:
did you verify the test fails without your change and then goes green with your change? I often do that to avoid false positives!

Reply:
yeah i usually do this but today I was being lazy :( let me break it then put this back

Reply:
revert my change and verified that it failed